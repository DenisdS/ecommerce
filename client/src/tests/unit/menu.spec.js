import renderer from 'react-test-renderer';
import React from 'react';
import Menu from '../../components/menu'

it('Verify if menu renders correctly', () => {
  const tree = renderer
    .create(
      <Menu>
        <ul>
          <li><a></a></li>
          <li><a></a></li>
          <li><a></a></li>
          <li><a></a></li>
          <li><a></a></li>
        </ul>
      </Menu>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
