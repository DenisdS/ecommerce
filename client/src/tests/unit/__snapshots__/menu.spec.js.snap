// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify if menu renders correctly 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/todos"
        onClick={[Function]}
      >
        Todos
      </a>
    </li>
    <li>
      <a
        href="/exclusivos"
        onClick={[Function]}
      >
        Exclusivos
      </a>
    </li>
    <li>
      <a
        href="/promocoes"
        onClick={[Function]}
      >
        Promoção
      </a>
    </li>
    <li>
      <a
        href="/favoritos"
        onClick={[Function]}
      >
        Favoritos
      </a>
    </li>
  </ul>
</nav>
`;
